### PR TITLE
[fa] day 2 afternoon

### PR DESCRIPTION
**Pull request by alix1383**

### CODE REVIEW

**Comment by alix1383** [comment]
Can we now add Farsi to the [translations page](https://google.github.io/comprehensive-rust/running-the-course/translations.html) ?

https://github.com/google/comprehensive-rust/blob/main/TRANSLATIONS.md#incomplete-translations

@DannyRavi @javad-jafari

**Comment by DannyRavi** [comment]
> Can we now add Farsi to the [translations page](https://google.github.io/comprehensive-rust/running-the-course/translations.html) ?
> 
> https://github.com/google/comprehensive-rust/blob/main/TRANSLATIONS.md#incomplete-translations
> 
> @DannyRavi @javad-jafari

Maybe it can be placed in the incomplete parts

File: po/fa.po [hunk 1]
```diff
@@ -6189,41 +6224,52 @@ msgstr ""
...
+"توجه داشته باشید که Rust (هنوز) پشتیبانی از ویژه‌سازی را ندارد. به عنوان "
+"مثال، با توجه به `duplicate` اصلی، اضافه کردن یک پیاده‌سازی ویژه‌شده مانند "
+"`duplicate(a: u32)` نامعتبر است."
 
 #: src/generics/impl-trait.md:3
 msgid ""
 "Similar to trait bounds, an `impl Trait` syntax can be used in function "
 "arguments and return values:"
 msgstr ""
+"مشابه با محدودیت‌های trait، می‌توان از نحو `impl Trait` در آرگومان‌های تابع و "
```
Comment (DannyRavi) [inline po/fa.po:6271]: نحو -> no
syntax -> is better

File: po/fa.po [hunk 2]
```diff
@@ -6992,128 +7204,166 @@ msgid ""
...
 msgid "This section is long. Take a break midway through."
-msgstr ""
+msgstr "این بخش طولانی است. در میانه‌ی آن یک استراحت کنید."
 
 #: src/std-traits/comparisons.md:3
 msgid ""
 "These traits support comparisons between values. All traits can be derived "
 "for types containing fields that implement these traits."
 msgstr ""
+"این trait‌ها از مقایسه بین مقادیر پشتیبانی می‌کنند. همه‌ی این trait‌ها را می‌توان "
```
Comment (DannyRavi) [inline po/fa.po:7219]: please recheck this part.

**Review by DannyRavi — CHANGES_REQUESTED** [review]
please resolve some parts!

**Review by DannyRavi — APPROVED** [review]

**Pull request merged**